GameDev.sln

```diff
@@ -1,12 +1,10 @@
 ﻿
```

Comment:
waarom staat er een sln file in de commit. we zijn nu al meer dan 16 weken bezig.... GEEL